src/flow_solver/flow_solver_cases/periodic_turbulence.cpp

```diff
@@ -232,8 +334,11 @@ void PeriodicTurbulence<dim, nstate>::compute_unsteady_data_and_write_to_table(
                 << "    Energy: " << integrated_kinetic_energy
                 << "    Enstrophy: " << integrated_enstrophy;
     if(is_viscous_flow) {
-    this->pcout << "    eps_vorticity: " << vorticity_based_dissipation_rate
-                << "    eps_p+eps_strain: " << (pressure_dilatation_based_dissipation_rate + deviatoric_strain_rate_tensor_based_dissipation_rate);
+        this->pcout << "    eps_vorticity: " << vorticity_based_dissipation_rate
+                    << "    eps_p+eps_strain: " << (pressure_dilatation_based_dissipation_rate + deviatoric_strain_rate_tensor_based_dissipation_rate);
+    }
+    if(do_calculate_numerical_entropy){
+        this->pcout << "    Num. Entropy: " << std::setprecision(16) << numerical_entropy;
```

Comment:
```suggestion
        this->pcout << "    Num. Entropy: " << std::scientific << numerical_entropy;
```

Reply:
I'd rather not make this change -- I want to be able to see the very last digits of the numerical entropy printed to see if it's truly conserved.